Core/fft_mt_r2iq.cpp

```diff
@@ -200,42 +186,44 @@ void fft_mt_r2iq::Init(float gain, int16_t **buffers, float** obuffers)
 		}
 
 		filterplan_t2f_c2c = fftwf_plan_dft_1d(halfFft, pfilterht, filterHw[0], FFTW_FORWARD, FFTW_MEASURE);
+		float *pht = new float[halfFft / 2 + 1];
```

Comment:
float *pht = new float[**halfFft /4 + 1**];
We use a ht that is 1/4 of used FFT +1 it simplify convolution's implementation.